### PR TITLE
Weather App

### PR DESCRIPTION
**Pull request by Wusinho**
This project is a weather app using OpenWeather API to display the temperature and conditions of the desired city.

Built with

- HTML
- Bootstrap
- JavaScript
- Webpack
- Package.json
- OpenWeatherMap Api

### CODE REVIEW

**Review by MkrtichSargsyan — CHANGES_REQUESTED** [review]
## _Status:_ Changes Requested :recycle:

## Hello Juan Lazo Benza

THIS IS VERY IMPRESSIVE WORK. YOU HAVE DONE A GOOD JOB :+1:

## To Highlight :tada:


1. Your PR has a good title and description.
2. You have descriptive git commit messages.
3. You have mastered the git-flow


 There are some changes that you need to make in order to get your project approved

- [ ] It should have a toggle control to determine if the result will be shown in Celsius or Fahrenheit ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png))

- [ ] It should change the appearance based on the received data [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png)

- [ ] Kindly add a live version for the project


<hr>

## OPTIONAL

- [ ] It would be nice if you add a ‘loading’ component that displays from the time the form is submitted until the information comes back from the API [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/loading_component_requirement.png)

- [ ] You can fetch the current geolocation of the user when the app is first accessed and fetch the weather from that location

Please, make the REQUIRED changes and submit your request again! :grinning:
Good luck. 

> ps: . If you have any questions about my review or need help, you can message me on slack ( @mkrtichsargsyan )

File: src/main.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+body {
+  background-image: url("../assets/background.png");
+  // background-position: center ; /* Center the image */
```
Comment (MkrtichSargsyan) [inline src/main.scss]: Remove the commented code

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            },
+          },
+        ],
+      },
+    ],
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      //   appMountId: "app",
+      //   filename: "index.html",
```
Comment (MkrtichSargsyan) [inline webpack.config.js]: Kindly remove the commented code

File: README.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+This project is a weather app using OpenWeather API to display the temperature and conditions of the desired city.
+
+![screenshot](./assets/WeatherApp.png)
+
+## Live View
+
+<!-- [Live View](https://lazo-todolist.netlify.app/) -->
+
+## Built with
```
Comment (MkrtichSargsyan) [inline README.md]: Add a live version for the project

**Review by oloomoses — CHANGES_REQUESTED** [review]
## Hello @Wusinho,
**Well done on implementing some of the changes suggested by @MkrtichSargsyan :muscle: :clap:.**
## Highlights
- A user can search for a specific location :heavy_check_mark:
- Use of ES6 syntax :heavy_check_mark:
- Use of `async/await` :heavy_check_mark:
- Well done on the linter configuration :muscle:
- Toggle between C and F :heavy_check_mark: 

## To improve
- [x] Ensure that the app changes its appearance based on the received data ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))
- [x] Ensure the app is deployed and live link added in the readme as suggested in the previous review


## Optional
- [ ] It would be nice to work on the optional changes suggested by @MkrtichSargsyan

**Please make the above changes and ask for another review**

Feel free to reach me on slack or leave a comment while mentioning me and I will be happy to reply.
### Happy Coding
_Slack: Oloo Moses_

**Comment by Wusinho** [comment]
> ## Hello @Wusinho,
> **Well done on implementing some of the changes suggested by @MkrtichSargsyan muscle clap.**
> 
> ## Highlights
> * A user can search for a specific location heavy_check_mark
> * Use of ES6 syntax heavy_check_mark
> * Use of `async/await` heavy_check_mark
> * Well done on the linter configuration muscle
> * Toggle between C and F heavy_check_mark
> 
> ## To improve
> * [x]  Ensure that the app changes its appearance based on the received data ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))
> * [x]  Ensure the app is deployed and live link added in the readme as suggested in the previous review

There is a picture of every kind of weather, you need to try different locations to check it.

**Review by mahmoud717 — APPROVED** [review]
### STATUS: APPROVED ✔️ ✔️ 

**hi @Wusinho**

**Great job with making all of the requirements. the project is approved, you can go ahead and merge this pull request**

**Pull request merged**